setup.py

```diff
@@ -12,14 +14,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 from setuptools import setup
 
 
+CURR_DIR = os.path.abspath(os.path.dirname(__file__))
+
+with open(os.path.join(CURR_DIR, 'README.rst')) as file_open:
```

Comment:
I had an annoying issue recently that actually popped up in datacompy'd long description where it couldn't run in a specific environment (jupyter terminal) because it didn't have utf-8 as the default encoding, and it failed to decode.  I'd probably recommend being explicit (`open(filename, encoding='utf-8')`) to avoid that corner case

Reply:
Sounds good!